test/correctness/simd_op_check.cpp

```diff
@@ -1493,7 +1495,12 @@ struct Test {
 
         check("vadd(v*.b,v*.b)", hvx_width/1, u8_1 + u8_2);
         check("vadd(v*.h,v*.h)", hvx_width/2, u16_1 + u16_2);
-        check("vadd(v*.w,v*.w)", hvx_width/4, u32_1 + u32_2);
+        if (is_v62) {
+            // v62 - 32 bit saturating add
+            check("vadd(v*.uw,v*.uw):sat", hvx_width/4, u32_1 + u32_2);
```

Comment:
Again, there is no saturation in the above addition. For example, saturating a 16 bit value, say 'a', to an 8 bit unsigned value is equivalent to a_sat = clamp(a, 0, 255) i.e. a_sat = min(max(0, a), 255). So a saturating add has to saturate the result of the add.